Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1789,9 +1789,6 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         articles.map { article in
             guard let (documentation, title) = DocumentationContext.documentationNodeAndTitle(
                 for: article,
-                
-                // Articles are available in the same languages the only root module is available in. If there is more
-                // than one module, we cannot determine what languages it's available in and default to Swift.
```

Comment:
I feel like a version of this comment could stay, and indicate that this behavior is still the default, but that it can be overridden with the new directive.

Reply:
This comment was misplaced IMO, which is why I removed it. Instead, the comments in the diff below is what's most accurate and where the logic that chooses the available languages actually is.

Reply:
I think i see it. I don't see an explicit reference that an article defaults to the languages from the root module, though; just that the default language defaults to Swift if it's present.

Reply:
You're right! I added a back here to clarify. Thanks!